backend/menifest/redis-deployment.yaml

```
@@ -0,0 +1,57 @@
+            claimName: redis-pvc
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: redis-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi  # ✅ Persistent storage request (no `limits` allowed here)
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)